packages/kbn-cell-actions/src/actions/utils.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { KBN_FIELD_TYPES } from '@kbn/field-types';
+import { isBoolean, isNumber, isString } from 'lodash/fp';
+import { Serializable, SerializableArray } from '@kbn/utility-types/src/serializable';
+import { DefaultActionsSupportedValue, NonNullableSerializable } from './types';
+import { CellActionFieldValue } from '../types';
+
+export const SUPPORTED_KBN_TYPES = [
+  KBN_FIELD_TYPES.DATE,
+  KBN_FIELD_TYPES.IP,
+  KBN_FIELD_TYPES.STRING,
+  KBN_FIELD_TYPES.NUMBER,
+  KBN_FIELD_TYPES.BOOLEAN,
+];
+
+export const isTypeSupportedByDefaultActions = (kbnFieldType: KBN_FIELD_TYPES) =>
+  SUPPORTED_KBN_TYPES.includes(kbnFieldType);
+
+const isNonNullablePrimitiveValue = (
+  value: CellActionFieldValue
+): value is string | number | boolean => isString(value) || isNumber(value) || isBoolean(value);
+
+const isNonMixedTypeArray = (
+  value: Array<string | number | boolean>
```

Comment:
NIT:
```suggestion
  value: SerializableArray
```
?

Reply:
This function only receives  `Array<string | number | boolean>` because we call it after filtering out other values. We could use a more generic type like `SerializableArray` but the extra types won't be used. Also, If we receive a SerializableArray the type guard would have to be expanded to  `tring[]
  | number[]
  | boolean[]
  | null[]
  | undefined[]
  | SerializableArray
  | SerializableRecord[] `. I am not sure if it is worth it.